src/test/java/components/ParserTest.java

```diff
@@ -0,0 +1,158 @@
+package components;
+import command.AddCommand;
+import exception.NyabotException;
+import org.junit.jupiter.api.Test;
+import task.Task;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+
+    @Test
+    public void parseNewTask_todo_expectedBehavior() throws NyabotException {
```

Comment:
I like how the test method names follow the three part format in the coding standard. Good job!